Libs/MRML/Core/Testing/vtkMRMLTransformNodeTest1.cxx

```diff
@@ -10,10 +10,43 @@
 
 =========================================================================auto=*/
 
+#include "vtkMRMLBSplineTransformNode.h"
 #include "vtkMRMLCoreTestingMacros.h"
+#include "vtkMRMLScene.h"
 #include "vtkMRMLTransformNode.h"
 
 #include <vtkGeneralTransform.h>
+#include <vtkMatrix4x4.h>
+#include <vtkNew.h>
+#include <vtkTransform.h>
+
+bool Matrix4x4AreEqual(vtkMatrix4x4 *m1, vtkMatrix4x4 *m2)
```

Comment:
That's a good idea, I'll do that. I'll try to make it equivalent with ITK's fuzzy image to physical transform compare.